github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-periodics.yaml

```diff
@@ -68,3 +68,51 @@ periodics:
       - name: gcs
         secret:
           secretName: gcs
+- name: periodic-kubevirtci-bump-kubevirt
+  interval: 6h
```

Comment:
I know I asked  for running it 4 times, but considering how our PR script works, that would probably result in not being able to merge the PR. Lets change it back to 24 hours.

Reply:
Sure. Done.

Reply:
BTW I looked into how the `pr-creator` works. AIUI we only have a chance to stuff additional labels into the body :( But if we change the tool to support labels we might also be able to fix the missing `release-note/none` being required on the kubevirt PR. WDYT?

Reply:
Hm, the release-note issue could be fixed by writing into the PR description a

```release-note
NONE
```

block.

Reply:
Sure, but when we are going for auto approval, we might just be able to add that also. Something in the ways of:

`... --labels=lgtm,approved,release-note-none`